src/main/java/com/itacademy/cms/repository/UserToCourseDAO.java

```diff
@@ -0,0 +1,6 @@
+//package com.itacademy.cms.dao;
```

Comment:
please delete this and all commented code

Reply:
ok I did it